dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -13,58 +13,68 @@ toc::[]
 
 ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
 == Overview
-There are many ways to access the cluster. This section describes some
-commonly used approaches.
+There are many ways to access your pods in a cluster. This section describes
```

Comment:
Instead of `There are many ways to access your pods in a cluster. This section describes
some commonly used approaches.`
Maybe `This section describes some commonly used approaches to access your pods in a cluster. `